nixos/doc/manual/release-notes/rl-1609.xml

```diff
@@ -39,6 +39,9 @@ following incompatible changes:</para>
     <command>start</command>, <command>stop</command>,
     <command>restart</command>, <command>status</command>.</para>
   </listitem>
+  <listitem>
+    <para><literal>hardware.pulseaudio.configFile</literal> must now be available on evaluation time, previously it could be any file.</para>
+  </listitem>
```

Comment:
Ah, sorry, didn't notice that. But I was thinking specifically of the description of the `configFile` option.

It currently says:
>          The path to the configuration the PulseAudio server
>          should use. By default, the "default.pa" configuration
>          from the PulseAudio distribution is used.

This is rather a kind of base config before all other options managed by the nixos module. The description should reflect that. And it should maybe have the type `types.nullOr types.path`?

Should we maybe also use the standard `extraConfig` pattern here? That way when you have to add custom options you wouldn't have to take care of appending the default.pa from the pulseaudio store path if you're comfortable with those settings.